package.json

```diff
@@ -2018,19 +2001,15 @@
         "node-html-parser": "^1.1.13",
         "nyc": "^15.0.0",
         "playwright-chromium": "^0.13.0",
-        "plotly.js-dist": "^1.55.2",
```

Comment:
Hi there, I'm from the Plotly team! We would like this dependency to remain if possible, so that we can submit small PRs to your repos to get the Plotly version bumped without having to wait for other actions in intermediate repos :)